eq-author-api/repositories/OptionRepository.js

```diff
@@ -22,27 +24,48 @@ const checkForExistingExclusive = async answerId => {
 
 const findAll = (where = {}, orderBy = "id", direction = "asc") =>
   Option.findAll()
-    .where({ isDeleted: false, otherAnswerId: null })
+    .where({ isDeleted: false })
     .where(where)
     .orderBy(orderBy, direction);
 
 const getById = id => Option.findById(id).where({ isDeleted: false });
 
 const insert = async (
-  { label, description, value, qCode, answerId, mutuallyExclusive = false },
+  {
+    label,
+    description,
+    value,
+    qCode,
+    answerId,
+    mutuallyExclusive = false,
+    hasAdditionalAnswer
+  },
   trx = db
 ) => {
+  let additionalAnswer;
   if (mutuallyExclusive) {
     await checkForExistingExclusive(answerId);
   }
+  if (hasAdditionalAnswer) {
+    additionalAnswer = await Answer.insert(
+      {
+        properties: getDefaultAnswerProperties("TextField"),
```

Comment:
If you use `Answer.creaateAnswer` instead of `insert` this should be taken care of for you.